tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -277,9 +277,9 @@ class SomeClass extends \Foo\Bar\A implements \Foo\Bar\Izumi, A, \A\B, \Foo\Bar\
         yield 'catch in multiple namespaces' => [
             '<?php
 namespace {
-    try{ foo(); } catch (Exception $z) {}
-    try{ foo(); } catch (A\X $z) {}
-    try{ foo(); } catch (B\Z $z) {}
+    try{ foo(); } catch (\Exception $z) {}
```

Comment:
this is an edge case, but I belive `leading_backslash_in_global_namespace: true` should prepend the (FQ)CN with `\`

Reply:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.45.0/src/Fixer/Import/FullyQualifiedStrictTypesFixer.php#L432 should be `false ===` - I will commit fix shortly, introduced in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7459

Reply:
addressed in #7654